tools/Ecoregionalization_workflow/BRT_model.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="brt_analysis" name="BRT tool prediction" version="0.1.0+galaxy0" >
+    <description>for species distribution modelling</description>
+    <requirements>
+       <requirement type="package" version="4.3.0">r-base</requirement>
+       <requirement type="package" version="1.3_14">r-dismo</requirement>
+       <requirement type="package" version="2.1.8">r-gbm</requirement>
+       <requirement type="package" version="3.4.2">r-ggplot2</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/brt.R'
+         '$enviro'
+         '$species_files'
+         '$input3'
+         '$output1'
+         '$output2'
+         '$output3'
+         '$output4'
+    ]]></command>
+    <inputs>
+      <param name="enviro" type="data" format="txt,csv,tsv" label="Environment Data"/>
```

Comment:
Is txt a valid input?

Reply:
Yes, lthough test files are csv, this can be used with text files without any issues.